cobc/parser.y

```diff
@@ -18176,6 +18203,7 @@ _not_on_size_error:
 not_on_size_error:
   NOT_SIZE_ERROR statement_list
   {
+	$$ = cb_int2;
```

Comment:
as noted, this should not be needed, even if we enable the handler insertion as you've did